### DISCUSSION TITLE
With Swoole 5.1.2 which installation option should i use ? ./configure --with-postgres ? OR  ./configure --enable-swoole-pgsql ?

### DISCUSSION THREAD

**Discussion by fakharksa** [swoole/swoole-src#5338] [Q&A] [answered]
In previous versions, i was installing swoole from source code using option ./configure --with-postgres but now in the documentation i see new option ./configure --enable-swoole-pgsql. Which one should i use ? OR should i use both options ?

**Comment by leocavalcante** [comment, marked as answer]
Hi, @fakharksa. You should use `--enable-swoole-pgsql` for v5.1.2.

You can then check your installation with:
```shell
php --ri swoole | grep pgsql
```

The output should be:
```
coroutine_pgsql => enabled
```

**Comment by fakharksa** [comment]
Great thanks for the prompt response.
